test/createStore.spec.js

```diff
@@ -610,4 +611,65 @@ describe('createStore', () => {
       store.subscribe(undefined)
     ).toThrow()
   })
+
+  it('should implement Symbol.observable interop point', () => {
+    function foo(state = 0, action) {
+      return action.type === 'foo' ? 1 : state
+    }
+
+    function bar(state = 0, action) {
+      return action.type === 'bar' ? 2 : state
+    }
+
+    const store = createStore(combineReducers({ foo, bar }))
+
+    let key
+    if (typeof Symbol === 'function') {
+      if (Symbol.observable) {
+        key = Symbol.observable
+      } else if (typeof Symbol.for === 'function') {
+        key = Symbol.for('observable')
+      }
+    } else {
+      key = '@@observable'
+    }
+
+    expect(typeof store[key]).toBe('function')
+
```

Comment:
Here we can probably use `symbol-observable` now instead of having all of this code to figure out what to use.